crux/src/Crux.hs

```diff
@@ -278,7 +278,9 @@ runSimulator lang opts@(cruxOpts,_) =
                 mgt <- provedGoalsTree ctx' proved
                 case mgt of
                   Nothing -> return ()
-                  Just gt -> modifyIORef gls (Seq.|> gt)
+                  Just gt ->
+                    do reportStatus gt
```

Comment:
If there is more than one path (e.g., in the SplitDFS mode) we'll report the proof status separately for each path... probably it would be better to compile all the results and give a single report at the end?

Reply:
Yep, good point.

Reply:
Fixed in 6b3ca68